packages/aragon-wrapper/src/index.js

```diff
@@ -757,7 +766,7 @@ export default class Aragon {
    */
   async calculateTransactionPath (sender, destination, methodName, params, finalForwarder) {
     const finalForwarderProvided = this.web3.utils.isAddress(finalForwarder)
-    const defaultGasPrice = this.web3.utils.toWei('20', 'gwei') // TODO: Get from ethgasstation.info or another source for legit gas values
+    const defaultGasPrice = await this.getDefaultGasPrice()
```

Comment:
The gas price that we need to send could be a function of the gas limit that is set in the transaction (higher gas limit may require a higher gas price). We could make the gas the transaction will be sent with a parameter to the `defaultGasPrice` function.